src/m365/spo/commands/user/user-ensure.spec.ts

```diff
@@ -116,6 +217,82 @@ describe(commands.USER_ENSURE, () => {
     assert(loggerLogSpy.calledWith(ensuredUserResponse));
   });
 
+  it('ensures user for a specific web by loginName', async () => {
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `${validWebUrl}/_api/web/ensureuser`) {
+        return ensuredUserResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, webUrl: validWebUrl, loginName: validLoginName } });
+    assert(loggerLogSpy.calledWith(ensuredUserResponse));
+  });
+
+  it('ensures user for a specific web by entraGroupId', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groups/${validEntraGroupId}`) {
+        return groupM365Response.value[0];
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `${validWebUrl}/_api/web/ensureuser`) {
+        return ensuredGroupResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { verbose: true, webUrl: validWebUrl, entraGroupId: validEntraGroupId } });
+    assert(loggerLogSpy.calledWith(ensuredGroupResponse));
```

Comment:
Let's check the request body instead of the output of the command.

Reply:
Hi @milanholemans 
Sorry, I did not get this review comment. Can you please point me to any implementation of this?
Thank you in advance.

Reply:
I mean, we should check the contents of the request body, for example, something like this:
https://github.com/pnp/cli-microsoft365/blob/818fb5024a38596b6bd25490ff4b4a98a282869e/src/m365/spo/commands/tenant/tenant-site-unarchive.spec.ts#L122-L136